index.d.ts

```diff
@@ -12,6 +12,14 @@ Promise.race([]);
 
 pRace([]);
 //=> [RangeError: Expected the input to contain at least one item]
+
+pRace(signal => [
+	signal => fetch('/api', {signal}),
+	signal => setTimeout(10, {signal}),
+]);
+//=> Remaining promises other than first one will be aborted.
 ```
 */
-export default function pRace<ValueType>(iterable: Iterable<ValueType | PromiseLike<ValueType>>): Promise<ValueType>;
+type PromisibleIterable<ValueType> = Iterable<ValueType | PromiseLike<ValueType>>;
```

Comment:
```suggestion
type IterableOrPromise<ValueType> = Iterable<ValueType | PromiseLike<ValueType>>;
```

?

Reply:
IMHO, `IterableOrPromise` feels a little confusing to me since it gives the impression it would be like `Iterable<ValueType> | PromiseLike<ValueType>`, which is different from the actual type.

Reply:
Right. I misread the type. `PromisibleIterable` is grammatically and semantically incorrect though.